core/orchestrator.go

```diff
@@ -329,10 +326,9 @@ func (n *LivepeerNode) transcodeSeg(config transcodeConfig, seg *stream.HLSSegme
 	}
 	took := time.Since(start)
 	tProfileData := make(map[ffmpeg.VideoProfile][]byte, 0)
-	glog.V(common.DEBUG).Infof("Transcoding of segment %v took %v", seg.SeqNo, took)
+	glog.V(common.DEBUG).Infof("Transcoding of segment manifestID=%s seqNo=%d took %v", string(md.ManifestID), seg.SeqNo, took)
 	if isLocal && monitor.Enabled {
-		monitor.LogSegmentTranscodeEnded(seg.SeqNo, string(md.ManifestID), took,
-			common.ProfilesNames(md.Profiles))
+		monitor.SegmentTranscoded(0, seg.SeqNo, took, 0, common.ProfilesNames(md.Profiles))
```

Comment:
Would a separate function be more appropriate here and in `transcoder.go`? Also just want to double check that this won't lead to double counting successes if the B or any attached Ts also report success

Reply:
Removed one unused argument and added some explanation of calling logic 540ff3d586ae6f5aa706be8c4ab2c24c7f0176eb 
It will not lead to double counting, because each metric gets tagged with node name and node type, and in query we can aggregate by any tag.
If O is transcoding by itself, only call in `orchestrator.go` will report success, code in `transcoder.go` only reports when it sees real http urls, not local filenames (and that means it works as separate transcoder).